paper/paper.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'MNE-BIDS'
+tags:
+  - Python
+  - BIDS
+  - MNE
+  - neuroimaging
+  - MEG
+  - EEG
+  - iEEG
+authors:
+  - affiliation: 1
+    name: Stefan Appelhoff
+    orcid: 0000-0001-8002-0877
+  - affiliation: 2
+    name: Mainak Jas
+    orcid: 0000-0002-3199-9027
+  - affiliation: 3
+    name: monkeyman192
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 4
+    name: Teon L Brooks
+    orcid: 0000-0001-7344-3230
+  - affiliation: 5
+    name: Marijn van Vliet
+    orcid: 0000-0002-6537-6899
+  - affiliation: 6
+    name: Romain Quentin
+    orcid: 0000-0001-7659-3605
+  - affiliation: 7
+    name: Chris Holdgraf
+    orcid: 0000-0002-2391-0678
+  - affiliation: 8
+    name: Maximilien Chaumon
+    orcid: 0000-0001-9664-8861
+  - affiliation: 9
+    name: Ezequiel Mikulan
+    orcid: 0000-0001-7259-6120
+  - affiliation: 10
+    name: ktavabi
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 11
+    name: hoechenberger
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 12
+    name: Dominik Welke
+    orcid: 0000-0002-5529-1998
+  - affiliation: 13
+    name: Clemens Brunner
+    orcid: 0000-0002-6030-2233
+  - affiliation: 14
+    name: Alexander P Rockhill
+    orcid: 0000-0003-3868-7453
+  - affiliation: 15
+    name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
```

Comment:
@sappelhoff you should check your google hangouts :) anyway I'll say it here. Any objection to moving me here?

Also add @larsoner - he made significant contributions I'd say

Reply:
Fine by me if you think it's warranted -- Eric Larson, 0000-0003-4782-5360, Institute for Learning and Brain Sciences, University of Washington, Seattle, WA, USA